src/server.js

```diff
@@ -27,7 +27,7 @@ export const startServer = port => new Promise((resolve, reject) => {
   app.use(bodyParser.urlencoded({ extended: false }));
   app.use(cors());
 
-  app.use('/api/v1', indexRouter);
+  app.use(router);
```

Comment:
Implement @Simplemart17 feedback so we have something like this:
`app.use('/api/v1', router)`

Reply:
@Temmyogunbo Noted